frontend/src/components/WhatsNewModal.tsx

```
@@ -1,101 +1,98 @@
           <div className="flex items-center gap-2">
             <Sparkles className="w-5 h-5 text-primary" />
-            <h2 className="text-lg font-semibold">What&apos;s New</h2>
+            <div>
+              <h2 className="text-lg font-semibold">{headerText}</h2>
+              {commitSha && (
+                <span className="text-xs text-gray-400 dark:text-slate-500">{commitSha}</span>
+              )}
+            </div>
           </div>
           <button onClick={close} className="p-1 hover:bg-gray-100 dark:hover:bg-space-700 rounded-lg text-gray-500 dark:text-slate-400" aria-label="Close">
             <X className="w-5 h-5" />
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)